### DISCUSSION TITLE
[Self-hosted] Change Kong 8443 Port to something else.

### DISCUSSION THREAD

**Discussion by ogabelopes** [supabase/supabase#14178] [Questions]
Dear all,

I am in need of assistance with changing the port number for Kong from 8443 to a different port. The reason for this is that I have another service running on the default port, and I need to avoid port conflicts.

I have tried searching the docs for a solution, but unfortunately, I have not been successful in finding one that works for my particular situation. If anyone has experience with Kong and knows how to change the port number, I would greatly appreciate your help.

Thank you very much for your time and assistance.

**Comment by C5G6M7** [comment]
Hi,

I realize it has been a while and you may have figured this out already or moved on but just posting this for anyone who may run into the same issue. See my example setup here: https://github.com/orgs/supabase/discussions/27608

at the bottom of my example in the .env file code the KONG_HTTPS_PORT value can be changed to something else and this will change the port used on the host machine if needed to avoid conflicts. Note that inside the container the port will still be 8443 but that shouldn't matter as it wouldn't conflict with anything outside of the container, but if for any reason you did also want to change the port used inside the container you can change "8443/tcp" in the docker compose file to something else. Note that on this line it means that the host port is set according to the env variable KONG_HTTPS_PORT and the container port accessible from within the container is the second value after the the ":"

${KONG_HTTPS_PORT}:8443/tcp
